mantidproject/mantid
# Change the range in the color map editor => update selection widget min/max line edits

This fixes #12544 The VSI selection widget should react properly when you change the range of the color scale in the Paraview "color map editor". This is done here with a Vtk callback on the RGBPoints property of the Vtk proxy for the color lookup table of the color editor.

To test:
- Open the Paraview color map editor and use it to change the color scale range. Your changes should reflect in the VSI color selection widget (the minimum and maximum line edits).
- Play with colors and see if they behave consistently. I'm not totally sure if this is what we wanted to do, so let's chat about it (I'm putting this as "In progress" for now).

Release notes: See here http://www.mantidproject.org/ReleaseNotes_3_5_UI_Changes, in "3D Visualisation (VSI)".
